firstvoices/backend/search/indices/dictionary_entry_document.py

```diff
@@ -47,11 +49,20 @@ class Index:
 
 # Signal to update the entry in index
 @receiver(post_save, sender=DictionaryEntry)
-def update_dictionary_entry_index(sender, instance, **kwargs):
+def request_update_dictionary_entry_index(sender, instance, **kwargs):
+    if DictionaryEntry.objects.filter(id=instance.id).exists():
+        update_dictionary_entry_index.apply_async(
+            (instance.id,), countdown=2, link_error=link_error_handler.s()
```

Comment:
The `countdown` value here is required for most `post_save` signals to function on model creation. Although the function checks that the model exists before calling the async task, it takes a moment for the model to be created. This avoids the async task accessing a model that has not been fully created yet.

I'm open to alternatives/suggestions about this countdown length, but in testing this solution worked without errors.

Reply:
It looks like this is possible and occurring in some tests, I will look into a solution outside of the arbitrary wait time